src/main/webapp/WEB-INF/tags/base.tag

```diff
@@ -1,2 +1,5 @@
 <%@tag description="Common page content" pageEncoding="UTF-8"%>
+<div class="header-container">
 <h1><span class="blue">S</span><span class="red">n</span><span class="yellow">i</span><span class="blue">p</span><span class="green">p</span><span class="red">e</span><span class="yellow">t</span></h1>
```

Comment:
Please indent your HTML normally. I understand you can't put spaces between the spans, but you can still indent the h1 and a as normal.